ncm-hostsfile/src/main/perl/hostsfile.pm

```diff
@@ -218,7 +218,7 @@ sub Configure {
             if (exists $newncmhosts{$h}) {
                 $le = delete $newncmhosts{$h};
             } elsif (exists $ncmhosts{$h}) {
-                $le = delete $ncmhosts{$h};
+                next;
```

Comment:
shouldn't it also delete the entry from `$ncmhosts` to indicate it was processed? (i tried to read the code a bit, i'm not that familiar)